frontend/src/pages/Home.tsx

```diff
@@ -11,7 +11,7 @@ export default function Home() {
     return (
         <div className="homePage">
             <div className="title">
-                <h1>Getting bags</h1>
+                {/*<h1>Getting bags</h1>*/}
```

Comment:
Please do not commit commented out code unless necessary.

Reply:
ups, sorry you're right